Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -66,6 +70,10 @@ class IntroViewController: UIViewController, OnViewDismissable {
         setupLayout()
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        profile.prefs.setInt(1, forKey: PrefsKeys.IntroSeen)
+    }
+
```

Comment:
Moved this piece from the onViewDismissed closure to here. Figured it won't cause any harm - unless you spotted something 👀

Reply:
@nishant2718 this will change the current behavior to show the onboarding, we kept the previous flow where we set onboarding as seen only when the user closes or finishes the onboarding that's why  it was done in `onViewDismissed`. If we set the value here we could have cases were the user didn't finish the onboarding and won't see it again on next launch. If this change is intended disregard my comment

Reply:
Gotcha - will revert that one